Automattic/vip-go-mu-plugins
# Updated shared-plugins/two-factor

## Description
This PR aims to update the two-factor plugin to include the latest update which includes a vital security fix.

Release: [https://github.com/WordPress/two-factor/releases/tag/0.7.2](https://github.com/WordPress/two-factor/releases/tag/0.7.2)
ref: [90a86f7f452422ad34717c9f2fb945af0a01ac4d](https://github.com/WordPress/two-factor/commit/90a86f7f452422ad34717c9f2fb945af0a01ac4d)

## Changelog Description
### Updated the two-factor plugin
The latest reselease of two-factor includes a security fix.

## Checklist

## Steps to Test
1. Check out PR.
1. Go to `wp-admin` > `Plugins` > `Two Factor`
1. Verify version is 0.7.2
